flaml/nlp/autotransformers.py

```diff
@@ -1023,7 +1022,7 @@ def fit(
         self._set_num_labels(self.task_name)
         self._fp16 = fp16
         ray.shutdown()
-        ray.init(local_mode=ray_local_mode)
+        ray.init(local_mode=ray_local_mode, num_cpus=2)
```

Comment:
This constant constraint shouldn't be here. It prevents users to use more cpus or gpus.
You can move ray.shutdown and ray.init() out.